samples/java_layout/src/main/java/io/customer/android/sample/java_layout/sdk/CustomerIORepository.java

```diff
@@ -31,8 +33,15 @@ public void initializeSdk(SampleApplication application) {
         // TODO: Remove old builder and use new builder only to initialize the SDK
         // The new method should be called after the old method till the old method is removed
         // This is because the push and in-app modules are still using properties only initialized in the old method
-        CustomerIO.Builder oldBuilder = new CustomerIO.Builder(sdkConfig.getSiteId(), BuildConfig.API_KEY, application);
+        io.customer.sdk.android.CustomerIO.Builder oldBuilder = new io.customer.sdk.android.CustomerIO.Builder(sdkConfig.getSiteId(), BuildConfig.API_KEY, application);
```

Comment:
This and fetching token snippet below are only usages of old SDK instance in sample apps. All other calls should be routed to new instance.